app/models/user.rb

```diff
@@ -51,6 +51,7 @@ class User < ActiveRecord::Base
 
   before_validation :default_display_name, on: [:create, :update]
   before_validation :setup_unsubscribe_token, on: [:create]
+  before_validation :update_ouroboros_created, on: [:create]
```

Comment:
What's the plan to get this validation to run? I assume the first time they login and it updates the password? I am wondering about stargazing users and if they've already got a bcrypt password hash it may not run.